pages.hi/windows/winget.md

```diff
@@ -0,0 +1,36 @@
+# winget
+
+> विंडोज़ पैकेज प्रबंधक।
+> अधिक जानकारी: <https://learn.microsoft.com/windows/package-manager/winget>.
+
+- एक पैकेज इनस्टॉल करें:
+
+`winget install {{package}}`
```

Comment:
Please translate all the "package"s. ref: [common-arguments](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md)
```suggestion
`winget install {{पैकेज}}`
```